frontend/src/components/ccfriApplication/group/AddNewFees.vue

```diff
@@ -683,7 +687,12 @@ export default {
       'setNavBarValue',
       'setNavBarCCFRIComplete',
     ]),
-    addRow() {
+    addRow(arrayHasItems = false) {
```

Comment:
Since this isn't a generic function, there's no benefit to passing in arrayHasItems. We can just check CCFRIFacilityModel.dates directly in the function.